test/run-tests.py

```diff
@@ -49,7 +49,7 @@
     ],
     'wast2json': [
         ('RUN', '%(wast2json)s'),
-        ('ARGS', ['%(in_file)s']),
+        ('ARGS', ['%(in_file)s', '-o', '%(out_dir)s/out.json']),
```

Comment:
Is the out_dir unique for each test? I forget. If it isn't, then we'll probably want the `out.json` file to be based on the input filename so the tests won't be screwed up when run in parallel.

Reply:
It is unique yes